devel/capnproto/Portfile

```diff
@@ -52,6 +53,11 @@ if {[string match *clang* ${configure.cxx}]} {
     configure.ldflags-append    -stdlib=libc++
 }
 
+variant openssl description {Support TLS using the OpenSSL library} {
```

Comment:
Shouldn't we just always enable this? The BSD license doesn't have a problem with the OpenSSL license, and variants are always a hassle to maintain.

Reply:
That seems sane.

Reply:
This wasn't as much about licensing, as about enabling a lighter variant since the upstream provides this option. It shouldn't hurt to have it available by default, though! I'll push with this added to default_variants, then.